packages/EasyCodingStandard/src/Application/Application.php

```diff
@@ -44,6 +45,11 @@
     private $fixerFileProcessor;
 
     /**
+     * @var ErrorCollector
+     */
+    private $errorReporter;
```

Comment:
This naming confused me that you've added new `ErrorReporter` class.
Please keep class naming: `ErrorCollector = $errorCollector`